demos/motortwit/motortwit/utils.py

```diff
@@ -0,0 +1,65 @@
+import pytz
+import os
+import yaml
```

Comment:
I think you should run isort over codebase. Imports order is quite vary over the project.